### PR TITLE
Add development setup to README.md

### LINKED ISSUE #3: Update README.md
Add project info to the README.md
- [x] Developer setup instructions

### PR DESCRIPTION
**Pull request by no-reply**
Closes #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,25 @@ UCLA Library Management -- Tenejo
...
 -----------
 
+1. Clone the project repository:
+   `git clone https://github.com/UCLALibrary/samvera-mgmt.git; cd samvera-mgmt`
+1. Install dependencies
+   `bundle install`
 1. Setup your database.
    We use PostgreSQL. To support the test and development environments, you'll
    need have Postgres installed and running. In your `psql` console do
    `create role tenejo with createdb login`. Then do
```
Comment (hardyoyo) [inline README.md]: this is kinda pedantic, I know, but you should close your SQL statement with a semicolon, or it won't run

Comment (no-reply) [reply README.md]: :+1:

Comment (no-reply) [reply README.md]: Thanks @hardyoyo. I updated and force pushed the change.

**Comment by andrewbenedictwallace** [comment]
Added a bunch of notes from my less-prepared experience. It'd be nice if @hardyoyo and @no-reply could double-check, though, I still don't understand everything...

**Pull request merged**